.github/workflows/ci.yml

```
@@ -232,6 +234,8 @@ jobs:
           #make dist
           #rpmbuild -ta ./sandia-openshmem-*.tar.gz --define "configargs --with-ofi=${LIBFABRIC_INSTALL_DIR} ${{ matrix.sos_config }}" --define "_prefix /usr/shmem"
           #make clean
+          rm test
```

```suggestion
          # RPM build does not handle the symbolic link, so manually copy tests-sos/test:
          rm test
```
A comment here is probably helpful to guide whoever may need to generate an RPM in the future